adapters/appnexus/appnexus.go

```diff
@@ -379,13 +378,10 @@ func getMediaTypeForBid(bid *appnexusBidExt) (openrtb_ext.BidType, error) {
 }
 
 // getIabCategoryForBid maps an appnexus brand id to an IAB category.
-func (a *adapter) getIabCategoryForBid(bid *appnexusBidExt) (string, error) {
+func (a *adapter) findIabCategoryForBid(bid *appnexusBidExt) (string, bool) {
 	brandIDString := strconv.Itoa(bid.Appnexus.BrandCategory)
-	if iabCategory, ok := a.iabCategoryMap[brandIDString]; ok {
-		return iabCategory, nil
-	} else {
-		return "", fmt.Errorf("category not in map: %s", brandIDString)
```

Comment:
with new changes, we are losing this error statement. Can we handle the !ok as well?

Reply:
We are actually not using the error statement in the code except at this place. Neither there was a test case written for this. In the previous version of code, this error was not appended anywhere in the final response. So we should not keep it unless we use it. This particular function gets the iabCategory if present else should return empty string.